### ISSUE TITLE
Trajectory path in detail mode

### ISSUE THREAD

**Issue by Bretzel1** [studio-minus/ppg-bugs-and-requests#1142]
Basically just a line that shows the path that something you throw or is launched will take.

Useful for throwing grenades, blocks or whatever you can select.

**Comment by DindinYT37** [comment]
Please use the template. Also, kinda related to #965.

**Comment by 404CoderNotFound** [comment]
> Please use the template. Also, kinda related to #965.

Yeah, he made another request because zooi ignores a lot of good requests

**Comment by DindinYT37** [comment]
> > Please use the template. Also, kinda related to #965.
> 
> Yeah, he made another request because zooi ignores a lot of good requests

He probably didn't think of that request (they're two separate authors btw), anyway, it's not that zooi ignores a lot of good requests (since those also already have replies in most cases) - I'm somewhat sure it's just that zooi is unsure about the requests, though I cannot speak for zooi, this is just how I think.
